tests/test_hooks.py

```diff
@@ -108,6 +122,51 @@ def test_plugins_available(simple_plugin, simple_plugin_2):
     assert len(plugins) == 1
 
 
+def test_plugins_available_ignore_trash(datadir, simple_plugin, simple_plugin_2):
+    plugin_dirs = [simple_plugin["path"], simple_plugin_2["path"]]
+    hm = HookMan(specs=simple_plugin["specs"], plugin_dirs=plugin_dirs)
+
+    plugins = hm.get_plugins_available()
+    assert {p.id for p in plugins} == {"simple_plugin", "simple_plugin_2"}
+
+    hm._move_to_trash(datadir / "plugins", "simple_plugin")
+    plugins = hm.get_plugins_available()
+    assert {p.id for p in plugins} == {"simple_plugin_2"}
+
+    hm._move_to_trash(datadir / "plugins", "simple_plugin_2")
+    plugins = hm.get_plugins_available()
+    assert {p.id for p in plugins} == set()
+
+
+def test_try_clean_cache_ignore_os_errors(datadir, simple_plugin, monkeypatch):
+    import sys
+
+    win = sys.platform.startswith('win32')
```

Comment:
Please add a comment here explaining why some of the tests behavior is different on Windows (including the comments in the middle of the test)